hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientScheduledFutureProxy.java

```diff
@@ -63,10 +63,10 @@
 
     private ScheduledTaskHandler handler;
 
-    public ClientScheduledFutureProxy(ScheduledTaskHandler handler, ClientContext context) {
+    ClientScheduledFutureProxy(ScheduledTaskHandler handler, ClientContext context) {
         super(DistributedScheduledExecutorService.SERVICE_NAME, handler.getSchedulerName());
-        setContext(context);
         this.handler = handler;
+        setContext(context);
```

Comment:
I don't see this in the original PR. Is this needed?

Reply:
Yep, that was always there required for the parent `ClientProxy`.

Reply:
In 3.9, it was part of the super-constructor.